Peek-swiftUI/Modules/MainView/Views/WelcomeBackTopView.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+
+struct WelcomeBackTopView: View {
```

Comment:
make the view accepts the two text content

Reply:
done